libpsautohint/autohintexe.c

```diff
@@ -107,9 +108,27 @@ vstemCB(int right, int left, char* glyphName)
 }
 
 static void
-reportCB(char* msg)
+reportCB(char* msg, int level)
 {
-    fprintf(stdout, "%s", msg);
+    switch (level) {
+        case -1: /* LOGDEBUG */
+            if (debug)
+                fprintf(stdout, "DEBUG: %s\n", msg);
```

Comment:
If debug variable is constantly false, no debug logging message will be printed despite the level (unless i’m missing something)

Reply:
Old code (not shown in the diff) sets it. it used to be a local variable inside `main()` but it is global now.